pex/pip.py

```diff
@@ -126,7 +126,7 @@ def _calculate_env(
 
         if network_configuration.client_cert:
             assert not isolated
-            yield "PIP_CLIENT_CERT", network_configuration.cert
+            yield "PIP_CLIENT_CERT", network_configuration.client_cert
```

Comment:
(this conversation can be resolved if you're comfortable with it, but i'd like to know ask @Eric-Arellano whether he would find this particular proposed change useful)

Reply:
That seems over the top. If you wanted newtypes you could just `class CABundle(str): ...` right?

Reply:
Since this is the one and only consumer of cert / client_cert, I think even newtypes are over the top. I agree though that if this were exposed to users in an API a newtype would pull its weight and potentially prevent N flip-flops instead of the only possible one, which is right here.